### DISCUSSION TITLE
OSCAR page

### DISCUSSION THREAD

**Discussion by KZiemian** [oscar-system/Oscar.jl#3695] [General]
I just want to notice few things about page, maybe one of maintainers what to give it a look.

On [page](https://www.oscar-system.org/install/) the Julia 1.9.3 is called the newest release. I guess at one point update is needed.

**Comment by thofma** [comment]
Thanks. This will be fixed by https://github.com/oscar-system/oscar-website/pull/326. Feel free to open PRs if you find some typos/outdated stuff.
